pandas/core/groupby/generic.py

```diff
@@ -587,6 +587,12 @@ def nunique(self, dropna: bool = True) -> Series:
     def describe(self, **kwargs):
         return super().describe(**kwargs)
 
+    def count(self) -> Series:
+        return cast(Series, super().count())
+
+    def size(self) -> Series:
```

Comment:
This returns a DataFrame with `as_index=False`?

Reply:
Code above is in `SeriesGroupBy` class, and `as_index=False` is not valid when doing a group-by on a `Series`